react-app/src/components/CommonComponents/Navbar.js

```diff
@@ -6,16 +6,16 @@ function Navbar() {
   return (
     <NavBar collapseOnSelect fixed="top" variant="dark" expand="lg" className="navbar">
       <Container fluid>
-        <NavBar.Brand as={NavLink} to="/" className="navbar-brand">
+        <NavBar.Brand as={NavLink} to="/" className="navbar-brand" replace>
```

Comment:
DP1: What is the replace prop doing here?

Reply:
Ditto for all the other Nav Links

Reply:
I don't think it's completely necessary, but I added it because it prevents a warning in the console in a way that I don't think harms anything. With HashRouter, I was getting a warning saying "Hash history cannot PUSH the same path; a new entry will not be added to the history stack" and the `replace` prop makes it so that clicking the link replaces the route rather than push a new one (documentation [here](https://v5.reactrouter.com/web/api/Link/replace-bool)). Since there isn't any navigation we can do in this app besides clicking the routes themselves or a button that directly reroutes you somewhere, I figured it was safe to replace the entry (I don't know a case where it wouldn't be safe, so this is kind of an invented case and a best guess at what could go wrong). From looking at some issue threads, it doesn't look like that warning is harmful at all and is mostly intended to help in development (so you have an idea of why nothing might be happening), so I can definitely take out the `replace` flags, but there will be new warnings that pop up in the console (in development) occasionally.

Reply:
Very informative. No complaints. Full send 🚀